etna/records/migrations/0006_auto_20210709_0958.py

```diff
@@ -12,7 +12,7 @@ class Migration(migrations.Migration):
     operations = [
         migrations.AlterField(
             model_name="recordpage",
-            name="iaid",
+            name="metadataId",
```

Comment:
This needs to be undone - and a new migration created to account for the field change